src/python/bot/tasks/minimize_task.py

```diff
@@ -1194,19 +1191,16 @@ def _unset_dedup_flags():
   security_flag = crash_result.is_security_issue()
   if (security_flag != testcase.security_flag or
       state.crash_state != expected_crash_state):
-    logs.log_warn('Ignoring unrelated crash.\n'
+    logs.log_warn('Unrelated crash occurred:\n'
                   'State: %s (expected %s)\n'
                   'Security: %s (expected %s)\n'
                   'Output: %s\n' %
                   (state.crash_state, expected_crash_state, security_flag,
                    testcase.security_flag, state.crash_stacktrace))
-    return None, None
-
-  with open(output_file_path, 'rb') as file_handle:
-    minimized_keys = blobs.write_blob(file_handle)
-
-  testcase.minimized_keys = minimized_keys
-  testcase.put()
+    if not testcase.flaky_stack:
```

Comment:
I don't think this is ever set for engine tests. As far as I can see, it's only set after calling check_for_initial_crash, which is not called in the engine case.